.github/actions/setup-cli-deps/action.yml

```diff
@@ -15,18 +12,6 @@ runs:
         git config --global user.email "cli-foundations@shopify.com"
         git config --global user.name "CLI Foundations"
       shell: bash
-    - name: Install Ruby
-      if: ${{ inputs.needs-ruby == 'true' }}
-      uses: ruby/setup-ruby@679e23d4a31bcf7cd0a69acd348e5f79e8c6b443 # pin@v1
-      with:
-        ruby-version: ${{ env.RUBY_VERSION }}
-        bundler: 'latest'
-        bundler-cache: true
-    - name: Install Bundler dependencies
-      if: ${{ inputs.needs-ruby == 'true' }}
-      working-directory: packages/cli-kit/assets/cli-ruby
-      run: bundle install
-      shell: bash
```

Comment:
I just realized we probably still need those for the remaining CLI 2 commands. This might not be the right PR for this one.

Reply:
No, that config is for just for app related stuff, so we don't need it there. We have a different job for themes+ruby and we have the ruby dependency there still :)

Reply:
Oh which one is that? :D

Reply:
Sorry I'm not quite following how those run the `shopify theme` commands